app/views/components/summaryCardLSP.scala.html

```diff
@@ -71,14 +67,12 @@ <h3 class="app-summary-card__title">
         }
     </div>
     <footer class="app-summary-card__footer">
-        <div class="app-summary-card__actions">
-            <ul class="app-summary-card__actions-list">
+        <div class="app-summary-card__link">
             @{
                 if(!summaryCard.isAppealedPoint) {
                     if(summaryCard.isAddedPoint) {
-                            <li class="app-summary-card__actions-list-item">{messages("summaryCard.adjustedPoint.cannotAppeal")}</li>
+                            <span class="govuk-body">{messages("summaryCard.adjustedPoint.cannotAppeal")}</span>
```

Comment:
Should we use a span here as spans are phrasing content, or would `<div>` or `<p>` make more sense?

Reply:
Could you apply something like this?
https://github.com/hmrc/penalties-prototype/blob/f3675c609dc4a4ddccc920c88ab8be967dbb07d3/app/views/beta-baseline/penalties-scenario-adj.html#L97-L99

Reply:
Okay that works, doesn't seem to affect the screen reader either 👍